editor/src/clj/editor/tile_source.clj

```diff
@@ -540,7 +540,7 @@
             metrics (texture-set-gen/calculate-tile-metrics image-size properties collision-size)]
         (when metrics
           (merge properties metrics)))
-      (g/error-fatal "tile data could not be generated due to invalid values")))
+      (g/->error _node-id :image :fatal image-resource "tile data could not be generated due to invalid values")))
```

Comment:
When `calculateMetrics` returns null, we don't really know what the problem is. This would render the message to the scene, and would also stop the propagation of invalid tile-source-attributes. A more detailed validation message will be available on the properties pane.